requirements/requirements.insights.txt

```diff
@@ -181,7 +181,7 @@ pulp-ansible==0.7.1
     # via galaxy-ng (setup.py)
 pulp-container==2.3.1
     # via galaxy-ng (setup.py)
-pulpcore==3.10.0
+pulpcore==3.11.0
```

Comment:
probably need to run `make requirements` again as 3.11.0.post0 has been released

Reply:
@awcrosby done!